net/retriever/retriever.go

```diff
@@ -309,44 +309,44 @@ func (m *MessageRetriever) attemptDecrypt(ciphertext []byte, pid peer.ID, addr m
 	// Decrypt and unmarshal plaintext
 	plaintext, err := net.Decrypt(m.node.PrivateKey, ciphertext)
 	if err != nil {
-		log.Warning("Unable to decrypt offline message from %s: %s", addr.String(), err.Error())
+		log.Warning("[Offline Message] unable to decrypt cipher text to plain text from %s: %s", addr.String(), err.Error())
```

Comment:
> You accidentally included the precision change in this PR. Please remove. Also, the point of this improvement was to include the message ID in the log output so we can match it with the message ID from the sender and can prove positively that the message wasn't dropped because it's for another user.

I'm kind of confused by this request @placer14 @drwasho because isn't the message id and request tracked by the normal handleMessage handler? Why would this be special for offline messages?

Reply:
Sorry for ambiguity. I'm talking about including the offline message pointer (the CID) that we publish in the logs (here: https://github.com/OpenBazaar/openbazaar-go/blob/67437f493da098a8d458ca122a93c935d1454f68/core/net.go#L100) to also appear in the failures here so if we have both sets of logs we can confirm if these failures are true negatives (true negative: the message was intended for this peer AND the decrypting failed).